webapp/javascript/redux/reducers/continuous.ts

```diff
@@ -646,21 +759,39 @@ export const selectAppNames = (state: RootState) =>
 export const selectComparisonState = (state: RootState) =>
   state.continuous.comparisonView;
 
-export const selectIsLoadingData = (state: RootState) => {
+export const selectIsLoadingData = (state: RootState, pathname?: PAGES) => {
```

Comment:
Hmm, I don't quite get why this logic is necessary, can you explain bit more?

Here's what I am thinking:
Let's say I am in the singleView page, if cancellation works properly, nothing should be loading from the other pages. Therefore there's no make this distinction?

Reply:
>if cancellation works properly, nothing should be loading from the other pages

you are right, but image this case:
you are on Single View, you requests some data for this page and then go away to another page. At this moment "singleView data" thunk cancels with type `reloading` or `loading` in `state.continuous.singleView.type` . I wanted to reject this thunk with `failed` status, but if you go back to Single View page, you won't see any data until new request fulfilled. I've done this logic in this way just for user convenience.
![image](https://user-images.githubusercontent.com/7372044/173839956-0ce337be-1b16-44d5-9390-0db3760463ac.png)

 This explanation might be confusing, you can just take a look at `state.continuous.singleView.type` when you quit SingleView page and source code of SingleView page (there is logic based on loading/reloading/loaded status
![image](https://user-images.githubusercontent.com/7372044/173839376-347f01f5-3da9-4fd9-8a1c-0b5e56f8b027.png)


but in case when data failed/cancelled this page renders nothing. I think it's reasonable, so I didn't put `failed` status there